semantic.cabal

```diff
@@ -305,10 +304,12 @@ library
                      , semantic-tags ^>= 0
                      , semigroupoids ^>= 5.3.2
                      , split ^>= 0.2.3.3
+                     , streaming-attoparsec ^>= 1.0.0.1
                      , streaming-process ^>= 0.1
                      , stm-chans ^>= 3.0.0.4
                      , template-haskell ^>= 2.14
                      , time ^>= 1.8.0.2
+                     , utf8-string ^>= 1.0.1.1
```

Comment:
Our parser now works over `Word8` values, which is the right thing to do, but to convert to a `FilePath` we need a function of type `[Word8] -> String`. This provides that, and is faster than `Text.toString . Encoding.decodeUtf8 . B.pack`.